sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -229,30 +229,48 @@ export function getBooleanOrUndefined(value: any): boolean | undefined {
 }
 
 /**
- *  @ignore
- * Helper utility to check for and return JSON like object,
- * or undefined if not passed in. Treats empty string as `undefined`.
+ * @ignore
+ * Helper utility to check for and return valid JSON like object i.e.,
+ * a single JSON like object or array of JSON like objects.
  * @param value
  */
-export function getJSONOrUndefined(value: any): any | undefined {
-  if (value == undefined || (typeof value === "string" && value.trim() === "")) {
+export function getJSObjectOrUndefined(value: any): any | undefined {
```

Comment:
This new version doesn't do the white-space only strings to undefined conversion - is that intentional?

Reply:
Yes, since that was specific to feature/service driven constraints, moved it to the individual parameter handling which requires "" to be a valid JSON like object